packages/gradle-plugin/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/utils/DependencyUtilsTest.kt

```diff
@@ -184,8 +184,7 @@ class DependencyUtilsTest {
 
     // We need to make sure we have Maven Central defined twice, one by the library,
     // and another is the override by RNGP.
-    assertEquals(
-        2,
+    assertThat(2).isEqualTo(
         libProject.repositories.count { it is MavenArtifactRepository && it.url == repositoryURI })
```

Comment:
Can you invert this statement? `assertThat(...).isEqualTo(2)`

Reply:
Fixed